demos/using-dev-build/vite.plugins.ts

```diff
@@ -1,22 +1,17 @@
 import { readFileSync } from "fs";
 import { join } from "path";
 
-export const readCanisterIds = ({
-  prefix,
-  network,
-}: {
-  prefix?: string;
-  network: string | "ic" | "local";
-}): Record<string, string> => {
-  const canisterIdsJsonFile =
-    network === "ic"
-      ? join(process.cwd(), "canister_ids.json")
-      : join(process.cwd(), ".dfx", "local", "canister_ids.json");
```

Comment:
I think this is still needed for mainnet deployments of the app 😬

Reply:
Discussed offline. `network` added back to `vite.plugins`